docs/index.md

```diff
@@ -0,0 +1,22 @@
+# InGate
+
+## Overview
+
+InGate is an Gateway API and Ingress Controller for Kubernetes.
+
+Learn more about [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress) and [Gateway API](https://kubernetes.io/docs/concepts/services-networking/gateway) on the Kubernetes documentation site.
+
+## Get started
+
+## Community, discussion, contribution, and support
+
+While we are in transition, we are still using the Ingress NGINX Slack channels and community meetings.
+
+- Read [`CONTRIBUTING.md`](contributing/getting-started.md) for information about setting up your environment, the workflow that we expect, and instructions on the developer certificate of origin that we require.
+- Join our Kubernetes Slack channel for developer discussion: [#ingress-nginx-dev](https://kubernetes.slack.com/archives/C021E147ZA4).
```

Comment:
This isn't up-to-date anymore since you already changed it on `main` branch, right?

Reply:
rebased.